.cspell.json

```diff
@@ -1,6 +1,8 @@
 {
     "ignorePaths": [
         "graphics",
+        "src/assets/fonts",
```

Comment:
Does cpsell actually try to spellcheck the fonts? 😆

Reply:
Yes it does and I don't know why. Let me know if you have other solutions for this.
![Screen Shot 2022-05-05 at 3 03 50 PM](https://user-images.githubusercontent.com/99699764/166876298-482599a6-148d-43ce-a4f1-f51694767360.png)

Reply:
Looks like it's only spellchecking `.otf` files, so this could simply be `"*.otf"` in case we add actual code in `src/assets/fonts`.

Reply:
done